### DISCUSSION TITLE
Locale doesn't apply to user

### DISCUSSION THREAD

**Discussion by ArtemusKimen** [InseeFr/Keycloak-FranceConnect#80] [Q&A]
Hello,

First, thanks for your plugin FranceConnect!

It works great, but I have a problem with locale:

When a new user register using FranceConnect, user's locale is still 'en', instead of 'fr'.

1 - I create a new user using FrancConnect
2 - In console, I see the /authorize request with ui_locales=fr
3 - But when I check my user locale, it's still 'en'

Do you have any idea of what's is going wrong?

**Comment by lme-atolcd** [comment]
Hi @ArtemusKimen,

I hope you already found a solution :)
If it's not the case, can you check that you enabled l10n in your Keycloak realm ("Realm Settings" > "Localization") and that "français" is added to the supported locales list?
